modules/installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+// * installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-nutanix-failure-domains_{context}"]
+= Configuring failure domains
+
+Failure domains improve the fault tolerance of an {product-title} cluster by distributing control plane and compute machines across multiple Nutanix Prism Elements (clusters).
+
+[TIP]
+====
+It is recommended that three failure domains be configured to ensure high-availability.
+====
+
+.Prerequisites
+
+* You have an installation configuration file (`install-config.yaml`).
+
+.Procedure
+
+. Edit the `install-config.yaml` file and add the following stanza to configure the first failure domain:
++
+[source,yaml]
+----
+# ...
+platform:
+  nutanix:
+    failureDomains:
+    - name: <failure_domain_name>
+      prismElement:
+        name: <prism_element_name>
+        uuid: <uuid>
+      subnetUUIDs:
+      - <network_uuid>
+# ...
+----
++
+where:
+
+`<failure_domain_name>`:: Specifies a unique name for the failure domain. The name is limited to 64 or fewer characters, which can include lower-case letters, digits, and a dash (`-`). The dash cannot be in the leading or ending position of the name.
+`<prism_element_name>`:: Optional. Specifies the name of the Prism Element.
```

Comment:
maybe add more details on prismElement
is it name or uuid ? what is mandatory ? which takes precedence ?

Reply:
First- nice to meet you @tuxtof. Yanahua had commented on this separately. At the time you commented, I had forgotten to add a description for `<prism_element_uuid>`. Does this help to clarify things?

Reply:
Hey , nice to meet you too
you specify the name as optional but not the uuid
can we specify only the name ? if not what interest to specify name if uuid is mandatory ?